pybikes/bcycle.py

```diff
@@ -2,13 +2,25 @@
 # Copyright (C) 2023, Martín González Gómez <m@martingonzalez.net>
 # Distributed under the AGPL license, see LICENSE.txt
 
-from pybikes.gbfs import Gbfs
+from pybikes.gbfs import Gbfs, GbfsStation
 
 
 FEED_URL = "https://gbfs.bcycle.com/{uid}/gbfs.json"
 
+class BCycleStation(GbfsStation):
+
+    def __init__(self, info, * args, ** kwargs):
+        super(BCycleStation, self).__init__(info, * args, ** kwargs)
+
+        # similar as velib, ebikes are listed under num_bikes_available_types
+        # https://github.com/eskerda/pybikes/blob/master/pybikes/velib.py#L17-L20
+        self.extra['has_ebikes'] = True
```

Comment:
this could be conditional on `ebikes > 0` for each station. I'm actually not sure if all bcycle systems have ebikes.